internals/cli/cli.go

```diff
@@ -268,6 +269,7 @@ func (e *exitStatus) Error() string {
 type RunOptions struct {
 	ClientConfig *client.Config
 	Logger       logger.Logger
+	PebbleDir    string
 }
```

Comment:
This looks fine to me. My only question would be since we are now effectively setting the Daemon server socket based on ```ClientConfig.socket```, is this OK ? Should we document this (for now its only derivative projects that use ```Run(...)``` with non-nil options that needs to know this), or would it be a bit nicer to add another ```ServerSocket``` to the ``RunOptions``` struct to make this a bit clearer?

Reply:
Good idea to add a clarifying comment here.

Reply:
added:

```diff
 type RunOptions struct {
+       // when starting a daemon ("pebble run"), the ClientConfig.Socket value
+       // is also used as the Unix domain socket path for the Pebble socket
```